packages/geofire-common/src/index.ts

```diff
@@ -29,6 +29,10 @@ export const E2 = 0.00669447819799;
 // Cutoff for rounding errors on double calculations
 export const EPSILON = 1e-12;
 
+export type geopoint = [number, number];
```

Comment:
If we're going to export these I think they should have capitalized names like `GeoPoint`, `GeoHash`, `GeoHashRange`.  Lowercase type names are normally reserved for actual "primitives"

Or you could just not export them. The compiler can see them anyway.

Reply:
Good one. I renamed them to Geopoint, Geohash, and GeohashRange. Following common naming convention for Geohash (see: https://en.wikipedia.org/wiki/Geohash), and then Geopoint to follow suit.